README.md

```diff
@@ -11,7 +11,7 @@ Tested with the following apps:
 
 ## Enabling TOTP 2FA for your account
 ![](https://raw.githubusercontent.com/owncloud/twofactor_totp/stable9.1/screenshots/settings.png)
-![](https://raw.githubusercontent.com/owncloud/twofactor_totp/stable9.1/screenshots/verify.png)
+![](https://raw.githubusercontent.com/owncloud/twofactor_totp/stable9.2/screenshots/verify.jpg)
```

Comment:
The reference to `stable9.2` does not look correct. How will this work after merge of this PR?

Reply:
Just checked if the link (old and new) returns an image but only does a 404